### ISSUE TITLE
unhandled error when setting title

### ISSUE THREAD

**Issue by txoof** [txoof/querylms#5]
Under some conditions, querylms will throw this attribute error when setting the title. 

```File [~/src/PaperPi/venv_paperpi-9876705927/lib/python3.11/site-packages/QueryLMS/QueryLMS.py:772](http://192.168.1.174:8888/lab/tree/PaperPi/paperpi/plugins/lms_client/PaperPi/venv_paperpi-9876705927/lib/python3.11/site-packages/QueryLMS/QueryLMS.py#line=771), in QueryLMS.get_now_playing(self)
    769 now_playing = self._add_keys(now_playing, True)
    771 if now_playing.get('remote'):
--> 772     now_playing['title'] = now_playing.get('remoteMeta', now_playing.get('title', '')).get('title', '')
    774 # ensure there is always an album_id value
    775 if not now_playing.get('album_id', False):

AttributeError: 'str' object has no attribute 'get'
```

**Comment by txoof** [comment]
this is the result of not properly sanitizing incoming dictionary data. In some instances the `remote` key that is returned by the status query is an int and other times is a string.

convert the value to a more truthy value before checking for remote streams.

**Comment by txoof** [comment]
Resolved in version 0.2.5.0 release
